sanic/static.py

```diff
@@ -93,6 +98,17 @@ def register(app, uri, file_or_directory, pattern,
                     headers=headers,
                     content_type=guess_type(file_path)[0] or 'text/plain')
             else:
+                if stream_large_files:
```

Comment:
I think since this is just affecting static files doesn't necessarily this should affect the side of performance we necessarily care about